executor/pom.xml

```diff
@@ -42,10 +42,6 @@
       <groupId>org.kie</groupId>
       <artifactId>kie-dmn-feel</artifactId>
     </dependency>
-    <dependency>
-      <groupId>io.quarkus</groupId>
-      <artifactId>quarkus-resteasy-qute</artifactId>
-    </dependency>
```

Comment:
It seems this module has more unsused dependecies. What is your opinion, as we touch the module, maybe we can do some more compreyhensive cleanup? Or should we do it separately? Or is the `mvn dependency:analyze` report misleading? 
```
[WARNING] Unused declared dependencies found:
[WARNING]    io.quarkus:quarkus-arc:jar:2.7.0.Final:compile
[WARNING]    io.quarkus:quarkus-micrometer-registry-prometheus:jar:2.7.0.Final:compile
[WARNING]    io.quarkus:quarkus-smallrye-reactive-messaging-kafka:jar:2.7.0.Final:compile
[WARNING]    io.quarkus:quarkus-smallrye-health:jar:2.7.0.Final:compile
[WARNING]    io.quarkus:quarkus-container-image-jib:jar:2.7.0.Final:compile
```

Reply:
I would rather keep that separate, feel free to open a JIRA/PR

Reply:
Ok I reported https://issues.redhat.com/browse/MGDOBR-487

Reply:
Please create us a JIRA to fix it @jomarko 👍